new-frontiers-14/frontier-station-14
# Adds a new condiment station

## About the PR



This PR adds a new dispenser for condiments, in addition to a dispenser for condiment cups and crafting means for each.

Big shout to @wallflowerghost for the sprite glowup.

## Why / Balance


I was looking for an alternative approach to the condiment station vending machine and thought this might be a viable alternative. PLUS I needed a way to offload 1000u of ketchup, so having a way to better get the ketchup out of the barrel and into the customer's hands, even 5u at a time, seemed ideal.

## How to test


Place and ensure the filled condiment machine comes stocked as desired.
Ditto condiment cups.
Ensure that all crafting recipes are functioning as desired.

## Media

![ss+(2024-12-30+at+10 47 00)](https://github.com/user-attachments/assets/56337667-403f-4db9-98b7-fc2cbc77793d)

![ss+(2024-12-30+at+11 32 48)](https://github.com/user-attachments/assets/85543e16-dfe7-45d9-8e7e-c25150e83f6b)


## Requirements



## Breaking changes


**Changelog**


🆑 
- add: Condiment dispenser.
